hasjob/views/listing.py

```diff
@@ -162,13 +160,31 @@ def jobdetail(domain, hashid):
 
     is_bgroup = getbool(request.args.get('b'))
     headline = post.headlineb if is_bgroup and post.headlineb else post.headline
+    is_post_admin = is_siteadmin or post.admin_is(g.user) or (g.user and g.user.flags.is_employer_month)
```

Comment:
Okay, then maybe a method called `is_accessible_to_user` on `JobPost`? Just that, this seems like an important rule that is in use at multiple points. Makes sense to me to abstract it out.